_studio/mfx_lib/cmrt_cross_platform/include/cmrt_cross_platform.h

```diff
@@ -1599,6 +1629,7 @@ namespace CmLinux
         CM_RT_API virtual INT SetVmeSurfaceStateParam(SurfaceIndex* pVmeIndex, CM_VME_SURFACE_STATE_PARAM *pSSParam) = 0;
 
         CM_RT_API virtual int32_t GetVISAVersion(uint32_t& majorVersion, uint32_t& minorVersion) = 0;
+        CM_RT_API virtual int32_t CreateQueueEx(CmQueue *&pQueue, CM_QUEUE_CREATE_OPTION QueueCreateOption = CM_DEFAULT_QUEUE_CREATE_OPTION) = 0;
```

Comment:
New virtual function? Which CMRT version introduce it?

Reply:
It is in CMRT pretty long time, was not reflected in our tables.